src/Microsoft.SqlTools.ServiceLayer/Formatter/TSqlFormatterService.cs

```diff
@@ -103,6 +110,14 @@ private static TelemetryProperties CreateTelemetryProps(bool isDocFormat)
 
         private async Task<TextEdit[]> FormatRangeAndReturnEdits(DocumentRangeFormattingParams docFormatParams)
         {
+            if (docFormatParams == null
+                || docFormatParams.TextDocument == null
+                || docFormatParams.TextDocument.Uri == null
+                || ShouldSkipFormatting(docFormatParams.TextDocument.Uri))
+            {
+                return Array.Empty<TextEdit>();
```

Comment:
I'm not very familiar with this way of instantiating an empty array. Can't you do it with empty `[]`?

Reply:
You can do it with `new TextEdit[0]`. I just prefer this way as it has nice clarity in my opinion.